### ISSUE TITLE
Typo in copying of cell-state?

### ISSUE THREAD

**Issue by leox1v** [schelotto/Skim_RNN_PyTorch#9]
`            c_tilde = torch.transpose(torch.stack(
                            [c_l_,
                             torch.cat([c_s[:, :self.small_cell_size],
                                        c_l_[:, self.small_cell_size:self.large_cell_size]],
                                       dim=1)
                             ], dim=2), 1, 2)`

Shouldn't it be `c_l[:, ...]` here instead of `c_l_[:, ...]`? The way it is right now it seems like you're using the large cell every time. Or am I missing something?
